custom-distributed-tracing/CustomDT.sln

```diff
@@ -0,0 +1,37 @@
+﻿
```

Comment:
We've been using a single solution file for all projects in the Examples repo (`NewRelicDotNetExamples.sln`). 

Could you add these projects to that solution instead of a stand-alone solution file please?

Reply:
Sorry, didn't realize.  Done.